airbyte-integrations/connectors/source-gitlab/metadata.yaml

```diff
@@ -29,6 +29,25 @@ data:
   releaseStage: generally_available
   releases:
     breakingChanges:
+      4.0.0:
+        message:
+          In this release, a format of stream state has been changed due to migration to low-code.
```

Comment:
In this release, several changes have been made to the Gitlab connector. The primary key...

Reply:
To be more clear

In this release, several changes have been made to the Gitlab connector. The primary key was changed for streams `group_members`, `group_labels`, `project_members`, `project_labels`, `branches`, and `tags`.

Reply:
Updated